package/network/config/ltq-vdsl-app/Makefile

```diff
@@ -58,9 +58,9 @@ CONFIGURE_ARGS += \
 #CONFIGURE_ARGS += --enable-model=debug
 
 define Package/ltq-vdsl-app/install
-	$(INSTALL_DIR) $(1)/etc/init.d $(1)/sbin
+	$(INSTALL_DIR) $(1)/etc/init.d $(1)/sbin $(1)/etc/hotplug.d/dsl
 	$(INSTALL_BIN) ./files/dsl_control $(1)/etc/init.d/
-	$(INSTALL_BIN) ./files/vdsl_cpe_control_wrapper $(1)/sbin/
+	$(INSTALL_BIN) ./files/10_atm_ptm.sh $(1)/etc/hotplug.d/dsl
```

Comment:
Please take care of the 10_atm_ptm.sh spit into 10_atm.sh + 10_ptm.sh here as well.